test/utest_sasview.py

```diff
@@ -3,11 +3,19 @@
 import subprocess
 import re
 import sys
+
+import logging
+import logging.config
+LOGGER_CONFIG_FILE = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'logging.ini')
+print LOGGER_CONFIG_FILE

```

Comment:
`print` usage should be as a function, not a statement. i.e. `print(LOGGER_CONFIG_FILE`.